5.5/Dockerfile.rhel7

```diff
@@ -13,17 +13,18 @@ FROM rhel7.2
 ENV MYSQL_VERSION=5.5 \
     HOME=/var/lib/mysql
 
-LABEL io.k8s.description="MySQL is a multi-user, multi-threaded SQL database server" \
+LABEL summary="MySQL is a multi-user, multi-threaded SQL database server" \
+      io.k8s.description="MySQL is a multi-user, multi-threaded SQL database server" \
       io.k8s.display-name="MySQL 5.5" \
       io.openshift.expose-services="3306:mysql" \
       io.openshift.tags="database,mysql,mysql55"
 
 # Labels consumed by Red Hat build service
-LABEL BZComponent="mysql55-docker" \
-      Name="openshift3/mysql-55-rhel7" \
-      Version="5.5" \
-      Release="1" \
-      Architecture="x86_64"
+LABEL com.redhat.component="mysql55-docker" \
```

Comment:
i don't really know the exact purpose of these labels, but it was mysql55-docker before, so if it was ok then, i'm ok leaving it.